src/handler/parsers.js

```diff
@@ -522,42 +49,778 @@ const SlashOptionsParser = async (options) => {
 
     return Alloptions;
 };
-const OptionParser = async (options, d) => {
-    const Checker = (msg) => options.includes(msg);
-    const optionData = {};
-    if (Checker("{edit:")) {
-        const editPart = options.split("{edit:")[1].split("}}")[0];
-        const dur = editPart.split(":")[0];
-        const msgs = editPart.split(":{").slice(1).join(":{").split("}:{");
-        const messages = [];
-        for (const msg of msgs) {
-            messages.push(await errorHandler(msg.split("}:{")[0], d));
+
+/**
+ * This function check if the given text is boolean true or false.
+ * @param {string} text - The text to check.
+ * @param {boolean} orelse - The boolean to return if its not boolean.
+ * @returns {boolean} - The result.
+*/
+let checkBoolean = (text, orelse) => {
+    if (orelse && typeof orelse !== "boolean") return;
+    if (!text  || typeof text   !== "string" ) return orelse;
+
+    if ( text && text.trim() && (text.trim() === "" || text.trim() === " ") ) 
+        return orelse;
+
+    return text?.trim()?.toLowerCase() === "true" ? true : (text?.trim()?.toLowerCase() === "false" ? false : orelse);
```

Comment:
Can be replaced with Boolean() I assume? No need for that as yes/no support is gone.